### PR TITLE
Standardized comments and separators in ffmpeg, gdal and klv.

### PR DESCRIPTION
**Pull request by gbrooksby**

### CODE REVIEW

File: arrows/gdal/image_io.h [hunk 1]
```diff
@@ -25,23 +23,23 @@ class KWIVER_ALGO_GDAL_EXPORT image_io
 public:
   // No configuration for this class yet
   /// \cond DoxygenSuppress
-  virtual void set_configuration(vital::config_block_sptr /*config*/) { }
-  virtual bool check_configuration(vital::config_block_sptr /*config*/) const { return true; }
+  virtual void set_configuration(vital::config_block_sptr //config
+) { }
+  virtual bool check_configuration(vital::config_block_sptr //config
+) const { return true; }
```
Comment (daniel-riehm) [inline arrows/gdal/image_io.h:29]: This is something which should be looked at more closely

**Review by daniel-riehm — COMMENTED** [review]
LGTM apart from the inline C-style comment, which should probably be either removed entirely or kept as-is.

File: arrows/gdal/image_io.h (hunk 1 above)
Comment (gbrooksby) [reply arrows/gdal/image_io.h]: Thanks for catching this.  I've reverted to the original form. Though I'm not thrilled with that version, it wasn't the intent to catch things like this.

File: arrows/ffmpeg/ffmpeg_video_input.cxx [hunk 2]
```diff
@@ -131,26 +127,22 @@ class ffmpeg_video_input::priv
   // --------------------------------------------------------------------------
   priv() : m_klv_demuxer( m_klv_timeline ) {}
 
-  // ==================================================================
+  // --------------------------------------------------------------------------
```
Comment (mwoehlke-kitware) [inline arrows/ffmpeg/ffmpeg_video_input.cxx:131]: Any chance the scripts could remove blank lines like this? (IOW, between Doxygen and a separator line?)

File: arrows/ffmpeg/ffmpeg_video_input.cxx [hunk 3]
```diff
@@ -131,26 +127,22 @@ class ffmpeg_video_input::priv
...
 
-  /*
-   * @brief Whether the video was opened.
-   *
-   * @return \b true if video was opened.
-   */
+  //  @brief Whether the video was opened.
+  //
+  //  @return \b true if video was opened.
```
Comment (mwoehlke-kitware) [inline arrows/ffmpeg/ffmpeg_video_input.cxx:134]: Um... so this is already wrong, but it seems likely this was intended to be `///`, not `// `...

Comment (gbrooksby) [reply arrows/ffmpeg/ffmpeg_video_input.cxx:134]: Interesting.  It appears the script is replacing `/*` comment blocks with `//` and `/**` blocks with `///`.  I hadn't noticed this before.

**Review by daniel-riehm — APPROVED** [review]
LGTM

**Pull request merged**